Add Mapping and Modding Timeline article (#4050)
ppy/osu-wiki: wiki/Mapping_and_Modding_Timeline/en.md

- **2019-03-13:** Rule requiring consecutive nominations to be set at least 24 hours apart from each other is removed. <!-- https://osu.ppy.sh/community/forums/posts/6885033 -->

### May

- **2019-05-05:** The BN/QAT restructure is finalized. The Quality Assurance Team becomes the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team), while the Beatmap Nominators become responsible for disqualifications and veto mediation. The [Beatmap Nominator website](https://bn.mappersguild.com/) was published to automate many BN/NAT tasks.

### August

- **2019-08-05:** [Naxess](https://osu.ppy.sh/users/8129817) releases the Mapset Verifier <!-- TODO: stub --> tool, automating a significant portion of beatmap checking processes for Beatmap Nominators.
- **2019-08-06:** Beatmap Nominators are able to participate in BN application evaluations. Their input is used to determine applicants and to judge potential new NAT additions.

### October

> - **2018-10-22:** [Ascendance](https://osu.ppy.sh/users/2931883) becomes a NAT leader after [Mao](https://osu.ppy.sh/users/2204515) steps down.

Reviewer: this type of thing is good to document as history ofc but I feel like it also needs to be connected to other points in this document or at least commented on why it's significant. (for this line) what did Ascendance do as NAT leader? did he have an impact on modding and mapping worth writing down? this doesn't tell reader anything important and it's not easy to discover answers to these questions if you weren't aroudn at the time

    - The `flame` (![flame icon](/wiki/shared/icon/flame.gif)) icon discontinued use. <!-- https://osu.ppy.sh/community/forums/topics/631077?start=6050796 -->

### September

- **2017-09-10:** The "[QAT Upheaval](https://osu.ppy.sh/community/forums/topics/635507)" was implemented. This was a series of changes in response to dissatisfaction from the mapping community under leadership by [osu!team](/wiki/People/The_Team) higher-ups. Following this change, the QAT began self-leadership without higher-up intervention.
    - BN tiers were replaced with [Probationary Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators#probationary-beatmap-nominators). Two probationary BN members could not nominate the same beatmap and users could not be in Probation for more than two months at a time.
    - Beatmap Nominators were given purple titles on the forums.
    - [Okoratu](https://osu.ppy.sh/users/1623405) and [JBHyperion](https://osu.ppy.sh/users/4879508) were appointed as QAT leaders <!-- TODO: stub for QAT Leaders --> by a vote of BN/QAT members. Their responsibility was to work with the osu!team to maintain the mapping and modding communities. 

>     - *Elite Nominator* profile badges and forum titles were rewarded to exceptional Beatmap Nominators.

Reviewer 2: uhhhh, pretty sure this was at the end of 2019, I never heard of anything similar happening in 2017.

    - [Featured Artist](/wiki/Featured_Artists) beatmaps became more frequently created and ranked.
    - `#mappers-guild` channel was created in the [osu!dev Discord server](/wiki/osu!dev_Discord_server).
- **2019-03-13:** Rule requiring consecutive nominations to be set at least 24 hours apart from each other was removed.
    - This rule was a major complaint from mappers and Beatmap Nominators because it felt like it was unnecessarily wasting everyone's time. If an issue was found after the first nomination, the second nominator would often avoid pointing it out to avoid an extra 24 hour wait.
    - Along with this change, Beatmap Nominators could veto qualified beatmaps. The rule originally existed because Beatmap Nominators needed 24 hours to potentially apply vetoes to maps with one nomination.<!-- https://osu.ppy.sh/community/forums/posts/6885033 -->

### May

- **2019-05-05:** The BN/QAT restructure was finalized.
    - The Quality Assurance Team became the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).
    - Beatmap Nominators became responsible for disqualifications and veto mediation.

>     - The [Beatmap Nominator website](https://bn.mappersguild.com/) was published to automate many BN/NAT tasks. <!-- https://osu.ppy.sh/home/news/2019-05-05-bn-and-qat-restructure -->

Reviewer 2: At this time, BN applications changed from being cycle based to allowing users to apply at any time, with individualized cooldown periods if they did not get in or were removed. This is a pretty big shift from the previous large and occasional waves being replaced with new bns being added in small amounts more regularly.

Besides saving the NAT from spreadsheet insanity, I think that's one of the bigger changes the website offered for the community view.

### August

- **2019-08-05:** [Naxess](https://osu.ppy.sh/users/8129817) released the Mapset Verifier <!-- TODO: stub --> tool, automating a significant portion of beatmap checking processes for Beatmap Nominators.
- **2019-08-06:** Beatmap Nominators became able to participate in BN application evaluations.

>     - Their input was used to determine applicants and to judge potential new NAT additions. <!-- https://github.com/pishifat/qat/commit/a8245a4c6fefcfe9368ffc1eee7e13ad6f8bb6af -->

Reviewer 2: Hmm, but BNs' evaluations don't play a very large role in this, so saying that they determine applicants (? applicants what) seems misleading

# Mapping and modding timeline

> Mapping and modding systems are constantly improving. The **Mapping and modding timeline** aims to document the biggest system changes and to serve as a reference tool when incorporating new changes.

Reviewer 3: I may be wrong, but "The timeline aims to serve as a reference when incorporating new changes" makes it sound like the timeline itself would make the changes.

    - Users could apply to learn about mapping and modding directly from volunteer MAT and BAT members.
    - The program was discontinued in 2013.<!-- https://osu.ppy.sh/community/forums/topics/34111 + https://osu.ppy.sh/community/forums/topics/34194 -->

### August

- **2010-08-27:** [ztrot](https://osu.ppy.sh/users/6347) was appointed as MAT leader after [Ephemeral](https://osu.ppy.sh/users/102335) stepped down. <!-- https://osu.ppy.sh/community/forums/topics/36208 -->
    - Leadership was tentatively passed between the two users until ztrot held control in February 2011. <!-- https://osu.ppy.sh/community/forums/topics/47118 -->

### October

- **2010-10-03:** MAT were granted permission to nominate beatmaps for ranking with normal `bubble` (![bubble icon](/wiki/shared/icon/bubble.gif)) icons, as opposed to proto-bubbles.
    - Proto-bubbles were ignored by most BAT members, so the MAT had no purpose without this change.
    - This gave MAT considerably more influence in the ranking process.<!-- https://osu.ppy.sh/community/forums/topics/38403 -->
- **2011-10-10:** MAT members were no longer allowed to nominate or vote on new additions to their team.

>     - Having gained bubbling permissions, it was assumed that the MAT would vote with circlejerk-related reasons, so this was prevented before damage could be done.

Reviewer 3: uhh, I am sure we can do without referencing group masturbation here

**1327 beatmaps** were ranked in 2013.

## 2014

### February

- **2014-02-10:** To improve organization, mapping and modding system internal management shifted from the forums to Trello. <!-- https://osu.ppy.sh/community/forums/topics/185379 -->
- **2014-02-18:** [Approved](/wiki/Beatmaps#approved) beatmap category was retired.
    - Marathon beatmaps were promoted to Ranked with 3 BAT nominations (2 `bubble` (![bubble icon](/wiki/shared/icon/bubble.gif)) icons + 1 `flame` (![flame icon](/wiki/shared/icon/flame.gif)) icon). <!-- date is not accurate. this is date of last approved map's approval -->

<!-- some time around here p3n becomes the equivilent of BATmanager but there is no documentation. probably on Trello -->

### August

- **2014-08-21:** The [Quality Assurance Team](/wiki/Modding/Quality_Assurance_Team) (*QAT*) splits off from the current BAT.
    - The BAT (often called newBAT at this time) nominated beatmaps for rank.
    - The QAT ensured qualified beatmaps were sufficient quality. Beatmaps that did not meet quality standards were disqualified.
    - This essentially reversed the merging of the BAT and MAT last year.

>     - The split was implemented as part of a manual trial for [moddingv2](/wiki/Beatmap_Discussion) attempting to automate BAT promotion and retirement. <!-- https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat -->

Reviewer 3: ```suggestion
    - The split was implemented as part of a manual trial for [modding v2](/wiki/Beatmap_Discussion) attempting to automate BAT promotion and retirement. <!-- https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat -->
```

    - Despite attempts at framing disqualifications as a positive change for mapping quality, the community viewed them negatively. This was especially apparent in cases of controversial beatmaps like the [disqualification on *Halozy - Genryuu Kaiko*](https://osu.ppy.sh/community/forums/posts/3591479)

### October

- **2014-10-15:** The [Beatmap Management subforum](https://osu.ppy.sh/community/forums/115) replaced Trello.

>     - Public accessibility to the BAT increased, which was the goal of moddingv2.

Reviewer 3: I keep stumbling on it, is the spelling intentional?
```suggestion
    - Public accessibility to the BAT increased, which was the goal of modding v2.
```

    - Public proposals on the Ranking Criteira subforum were no longer considered.
    - Taking a stance similar to the [Triumvir Conglomerate](/wiki/BAT_Managers#triumvir-conglomerate), the Ranking Criteria Council involved members to represent major mapping subgroups by region. Each game mode had 1 mapper and 1 player from the Americas, Europe, and Asia, along with 2 members of the QAT.
    - Each game mode's goal was to create a general/difficulty-specific Ranking Criteria for their respective mode. osu!catch was the only game mode that achieved this goal under this system.
    - The collective goal of this group was to rewrite all sections of the general Ranking Criteria. <!-- https://osu.ppy.sh/community/forums/topics/420229 -->
- **2016-02-24:** Mapset and spread restructure was introduced to limit the number of difficulties in each difficulty tear per beatmap set.
    - Negative public reception caused this to be rolled back immediately and its [forum thread](https://osu.ppy.sh/community/forums/topics/420223) accumulated 614 responses in 24 hours, making it osu!'s fastest moving thread.

>     - This attempt at undemocratic change caused the Ranking criteria council to immediately have a negative reputation. <!-- https://osu.ppy.sh/community/forums/topics/420223 -->

Reviewer 3: ```suggestion
    - This attempt at undemocratic change caused the Ranking Criteria Council to immediately have a negative reputation. <!-- https://osu.ppy.sh/community/forums/topics/420223 -->
```

    - The UBKRC's goals were the same as the Ranking Criteria Council, but it was more widely accepted because it was open to community influence.
    - Teams of experts were chosen for each subsection of the Ranking Criteria (e.g. one team of storyboarding experts, another team of skinning experts, etc.).
    - The group was managed by [Okoratu](https://osu.ppy.sh/users/1623405) and [pishifat](https://osu.ppy.sh/users/3178418).
    - The UBKRC's name was chosen because it was ridiculously stupid. Its aim was to appear less authoritative than the Ranking Criteria Council. <!-- https://osu.ppy.sh/community/forums/topics/480331 -->

### August

- **2016-08-05:** The [Quality Assurance Team blog](/wiki/Quality_Assurance_Team_Blog) was created.
    - The QAT was receiving criticism about being too closed-off from the rest of the mapping community, so this blog was created to increase transparency for QAT decision making and community activity. <!-- https://osu.ppy.sh/community/forums/topics/578437 -->
- **2016-08-26:** The [Code of Conduct](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) was implemented to establish modding behavior expectations.

>     - These rules and guidelines were treated as common sense up until this point. By documenting them, moderation could be more consistent.<!-- https://osu.ppy.sh/community/forums/topics/447557 -->

Reviewer 3: ```suggestion
    - These rules and guidelines were treated as common sense up until this point. By documenting them, moderation could be made more consistent.<!-- https://osu.ppy.sh/community/forums/topics/447557 -->
```

    - Beatmap Nominators could halt a beatmap in the ranking process for subjective reasons by resetting a nomination. This was done with a `bubble pop` (![bubble pop icon](/wiki/shared/icon/bubble-pop.gif)) icon.
    - Vetoes could be overwritten by the nominations of Beatmap Nominators previously uninvolved with the beatmap.
    - To allow adequate time to veto, consecutive nominations could not be set within 24 hours of each other.
    - Inconclusive vetoes were mediated by the QAT shortly after this system's implementation. <!-- date is not exact at all, but logs are from deleted discord chat -->
- **2016-11-02:** Beatmap Nominator files were created to keep tabs on all individual BN members.
    - These files included modding/nomination activity, disqualifications, and behavior issues.
    - Members of the BN with 3 or more "strikes" logged in these files would be removed from the group. <!-- QAT folder archives -->

**1720 beatmaps** were ranked in 2016.

## 2017

### April

- **2017-04-06:** [Beatmap Nominator Tiers](/wiki/Modding/Beatmap_Nominator_Tiers) were implemented, splitting the BN into two subgroups called "tiers".

>     - Because the QAT was no longer acting as a quality control gate, this system was implemented to improve the quality of beatmaps entering ranked.

Reviewer 3: ```suggestion
    - Because the QAT was no longer acting as a quality control gate, this system was implemented to improve the quality of beatmaps entering Ranked.
```

    - Because NAT leaders had often been passive towards negative decision making, Ascendance took an active controlling role and delivered bad news regarding NAT removals. <!-- source is discord logs -->

**3217 beatmaps** were ranked in 2019.

## 2020

### February

- **2020-02-07:** The first Elite Nominator profile badges were awarded to recognize continued exceptional performance from members of the BN team. 
    - These were originally planned in the QAT Upheaval two years prior. <!-- https://osu.ppy.sh/home/news/2020-02-07-community-contributors-2019 -->
- **2020-02-21:**  NAT leadership as a concept was discontinued as [Ascendance](https://osu.ppy.sh/users/2931883) stepped down as NAT leader.
    - Tasks specific to NAT leaders were delegated to the whole team.
- **2020-02-24:** The [Quality Assurance Helper](/wiki/People/The_Team/Beatmap_Nominators/General_Information#quality-assurance-helpers) system was transfered from Trello to the [Beatmap Nominator website](https://bn.mappersguild.com/).
    - This change was made to account for Quality Assurance activity in Beatmap Nominator evaluations.

### April

> - **2020-04-11:** The Mappers' Report <!-- TODO: stub --> led by [Feerum](https://osu.ppy.sh/users/4815717) was created to summarize mapping community events via front page news posts.

Reviewer 3: if you have chance AND if there was anything significant in the sphere of modding and mapping, please document it

Author: nothing's coming to mind
